client/flutter/assets/content_bundles/protect_yourself.en.yaml

```diff
@@ -0,0 +1,26 @@
+schema_version: 1
+content_version: 1
+contents:
+  type: fact
+  items:
+    -
+      animation_name: wash_hands
+      body_html: |
+        *Wash your hands* with soap and water to avoid getting sick and spreading infections to others.
+    -
+      image_name: avoid_touch
+      body_html: |
+        *Avoid touching* your eyes, mouth, and nose.
+    -
+      image_name: elbow
+      body_html: |
+        *Cover your mouth and nose* with your bent elbow or tissue when you cough or sneeze.
+    -
+      image_name: social_distance
+      body_html: |
+        *Stay more than* 1 meter (>3 feet) away from a person who is sick.
+    -
+      image_name: mask
```

Comment:
@hspinks Worth noting here also that the `protect_yourself` yaml contains plain text with `*` asterisks for bold rather than our regular HTML content.  This was to maintain consistency with the current page implementation but can be changed if desired.

Reply:
Think we should use just restricted html going forward and not roll our own less-than-markdown.  BUt its for a future PR :D

Reply:
#1029 #1030